.eslintrc.js

```diff
@@ -117,6 +116,12 @@ module.exports = {
         "no-unused-vars": 2,
       },
     },
+    {
+      files: ["src/**/*.tsx"],
+      rules: {
+        "react/no-multi-comp": ["error", { ignoreStateless: false }],
```

Comment:
I changed this to only `tsx` files as it was getting false positives from functions that return functions so hopefully this might help reduce that.